src/vivarium_public_health/population/base_population.py

```diff
@@ -202,6 +202,31 @@ def _validate_config_for_deprecated_keys(self) -> None:
                 f"Public Health. Use the new key '{mapper[key]}' instead."
             )
 
+    def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
+        return load_population_structure(builder)
+
+
+class ScaledPopulation(BasePopulation):
+    """Component for scaling the population structure based on a scaling factor."""
+
+    def __init__(self, scaling_factor: str | float | pd.DataFrame):
+        super().__init__()
+        self.scaling_factor = scaling_factor
+
+    def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
+        scaling_factor = self.get_data(builder, self.scaling_factor)
```

Comment:
Why this indirection? doesn't `get_data` just return the thing anyway unless it's a function?

Reply:
The most likely use-case is this is an artifact key, in which case we do need to call `get_data()` to get the actual data from the artifact.